extensions/myip/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# MyIP Changelog
+
+​
+
+## [Added Additions] - 2022-03-17
```

Comment:
```suggestion
# MyIP Changelog

## [Added Additions] - 2022-03-17
```
Seems like you have a hidden char on line three and some extra spaces, could you remove it? (So first entry starts on line 3 instead?